code/game/objects/objs.dm

```diff
@@ -31,8 +31,7 @@
 /obj/Destroy()
 	if (is_processing)
 		var/datum/controller/subsystem/processing/P = processing_subsystems_by_varname[is_processing]
-		if (P)
```

Comment:
what is going to happen if the subsystem isn't there without this check?

Reply:
If subsystem isn't there then we are doing something wrong and this should be fixed

Reply:
I think that P is useless. is_processing already contains enough info to STOP_PROCESSING